### DISCUSSION TITLE
utilities.css

### DISCUSSION THREAD

**Discussion by argyleink** [argyleink/open-props#75] [Ideas]
props that follow you between projects, generic yet always useful, maybe solve a wierd quark or provide good practices into a named variable.

**Comment by Afif13** [comment]
I don't know if they can fit here but I have a lot of ideas (using gradient as always 🤪) to have different kind of animation on hover.

Here is a demo of some underline effects: https://codepen.io/t_afif/pen/qBPbbWb

The idea is to define the animation within a background value and later have a "hover" variable to control the state (0 and 1). The hover variables can be shared between all the animations. 

----

I have more than 400 animations using only gradients 😅 https://dev.to/afif/100-underline-overlay-animation-the-ultimate-css-collection-4p40. Some of them require specific transition values but some of them not so I can easily grab a few of them to have a great collection.

**Reply by argyleink** [reply]
400! Needs it's own lib! The jit-props plugin would help

**Reply by Afif13** [reply]
Maybe we can have "effects.css" or "hover.css"?🤔 ... I will first try to group as much animation as possible then I let you decide about the organization.
